test/helpers/cdn.js

```diff
@@ -17,22 +17,29 @@ describe('cdn helper', function () {
         theme_version_id: '123',
         theme_config_id: '3245',
     };
+    var themeSettings = {
+      cdn: {
+        customcdn: {
```

Comment:
I definitely agree that the additional nesting is unnecessary, but I was thinking it would be better to start out with an object instead of a string in the event additional functionality like different endpoints based on environment. For example, Imgix can resize images based on URL parameters, but if you wanted to test this functionality locally you'd need to run an additional server, and setup something like this

```
cdn: {
    imgix: {
        path: "https://bc.imgix.net",
        local_path: "http://localhost:8080"
    }
}```

However, starting with using strings and then just adding a typecheck in the future if additional functionality is added could be a way around it as well! Just let me know what you'd prefer and I'll make it happen!

I'll also push a few additional tests with multiple endpoints configured.